app/src/renderer/js/main.js

```diff
@@ -41,6 +45,70 @@ document.addEventListener('DOMContentLoaded', () => {
 
   const [micOnIcon, micOffIcon] = toggleAudioRecordBtn.children;
 
+  const createOption = (label, disabled = false) => {
+    const option = document.createElement('option');
+    option.value = label;
+    option.text = label;
+    option.disabled = disabled;
+    return option;
+  };
+
+  const appData = {
+    Fullscreen: remote.screen.getPrimaryDisplay().bounds
+  };
+
+  function handleAppChange() {
+    const app = this.value;
+
+    if (!app) {
+      return;
+    }
+
+    if (app === 'Fullscreen') {
+      let {width, height} = appData[app];
+      // Need to get rid of the buffer because window can't be outside the screen limits
+      width -= cropperWindowBuffer;
+      height -= cropperWindowBuffer;
+      ipcRenderer.send('open-cropper-window', {width, height}, {x: 1, y: 1});
+    } else {
+      ipcRenderer.send('activate-application', app, appData[app]);
+    }
+  }
+
+  function clearApp() {
+    appSelector.querySelector('option[disabled]').selected = true;
+  }
+
+  appSelector.append(createOption('Fullscreen'));
+  appSelector.append(createOption('──────────', true));
+  appSelector.addEventListener('change', handleAppChange);
+
+  function loadApplications() {
+    // Remove existing applications
+    appSelector.querySelectorAll('option:not([disabled])').forEach(option => {
+      if (option.value !== 'Fullscreen') {
+        option.remove();
+        delete appData[option.value];
+      }
+    });
+
+    appSelector.querySelector('option[disabled]').text = 'Loading...';
+    appSelector.querySelector('option[disabled]').selected = true;
```

Comment:
You're using `appSelector.querySelector('option[disabled]')` 4 times. Save it to a variable.

Reply:
ea47eb8